src/reporting/clients/IMappingsClient.ts

```diff
@@ -0,0 +1,548 @@
+    mappingId: string,
+    groupId: string
+  ): Promise<Group>;
+
+  /**
+   * Updates a Group for a Mapping.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} accessToken OAuth access token with scope `insights:modify`.
+   * @param {GroupUpdate} group Request body.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/update-group/
+   */
+  updateGroup(
+    accessToken: AccessToken,
+    iModelId: string,
+    mappingId: string,
+    groupId: string,
+    group: GroupUpdate
+  ): Promise<Group>;
+
+  /**
+   * Deletes a Group for a Mapping.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} accessToken OAuth access token with scope `insights:modify`.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/delete-group/
+   */
+  deleteGroup(
+    accessToken: AccessToken,
+    iModelId: string,
+    mappingId: string,
+    groupId: string
+  ): Promise<Response>;
+
+  /**
+   * Gets all GroupProperties for a Group. This method returns the full list of group properties.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`.
+   * @param {number} top the number of entities to load per page.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-groupproperties/
+   */
+  getGroupProperties(
+    accessToken: AccessToken,
+    iModelId: string,
+    mappingId: string,
+    groupId: string,
+    top?: number
+  ): Promise<GroupProperty[]>;
+
+  /**
+   * Gets an async paged iterator of GroupProperties for a Group.
+   * This method returns an iterator which loads pages of group properties as it is being iterated over.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} accessToken OAuth access token with scope `insights:read`.
+   * @param {number} top the number of entities to load per page.
+   * @memberof ReportingClient
+   * @link https://developer.bentley.com/apis/insights/operations/get-groupproperties/
+   */
+  getGroupPropertiesIterator(
+    accessToken: AccessToken,
+    iModelId: string,
+    mappingId: string,
+    groupId: string,
+    top?: number
+  ): EntityListIterator<GroupProperty>;
+
+  /**
+   * Gets a GroupProperty for a Group.
+   * @param {string} iModelId The iModel Id.
+   * @param {string} mappingId The Mapping Id.
+   * @param {string} groupId The Group Id.
+   * @param {string} propertyId The GroupProperty Id.
+   * @param {string} accessToken access token with scope `insights:read`.
```

Comment:
Not capitalized - `Access token ...`, not `access token ...`

Reply:
Or even better - add the word `OAuth` so that is consistent with all the other methods.

Reply:
added the word OAuth